launch_local.sh

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash
```

Comment:
Maybe use dashes in the filename to be consistent with the other scripts. Also, some readme text to point to this script instead of what is there now would be nice.

Reply:
Just added a few lines in the README file.